src/lib/styled/styleFunctions.ts

```diff
@@ -49,8 +55,15 @@ transition: 200ms color;
   color: ${theme.disabledUiTextColor};
 }`
 
-export const sideBarTextColor = ({ theme }: StyledProps) =>
-  `color: ${theme.sideBarTextColor};`
+export const sideBarTextColor = ({
```

Comment:
the sidebarTextColor is already used in three components, and the other components have not been adapted to this change. 

keep only 
```
export const sideBarTextColor = ({ theme }: StyledProps) =>
  `color: ${theme.sideBarTextColor};`
```

or change the other affected components

Reply:
It is okey because UI is natural in other sections.